week04/ecommerce/Contact.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Us</title>
+    <link rel="stylesheet" media="screen and (max-width: 800px)" href="smallstyle.css">
+    <link rel="stylesheet" media="screen and (min-width: 800px)" href="fullstyle.css">
+</head>
+<header>
+    <!-- Navbar -->
+    <nav class="navbar">
+        <ul>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="products.html">Products</a></li>
+            <li><a href="contact.html">Contact</a></li>
+        </ul>
+    </nav>
+</header>
+
+<body>
+
+    <form name="contactPage" class="contactPage" action="#" method="#" onsubmit="return validateForm()">
```

Comment:
It may be a bit more legible to stack some of these labels/inputs instead of having them inline.

Reply:
Also i see no form validation which looks to be a requirement under the JS section of the directions.

Reply:
Also, to help with validation, i would look into the "required" attribute for your input tags. Can be super helpful and looks nice.